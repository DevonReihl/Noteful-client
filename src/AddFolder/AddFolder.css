
.AddFolder {
  color: #dedede;
}

.AddFolder h2 {
  margin-left: 16px;
}
