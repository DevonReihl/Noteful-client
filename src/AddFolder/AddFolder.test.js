import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AddFolder from './AddFolder'

describe(`AddItemForm component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<AddFolder />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
