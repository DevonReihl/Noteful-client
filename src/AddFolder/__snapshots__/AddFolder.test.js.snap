// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemForm component renders the complete form 1`] = `
<section
  className="AddFolder"
>
  <h2>
    Create a folder
  </h2>
  <NotefulForm>
    <div
      className="field"
    >
      <label
        htmlFor="folder-name-input"
      >
        Name
      </label>
      <input
        id="folder-name-input"
        type="text"
      />
    </div>
    <div
      className="buttons"
    >
      <button
        type="submit"
      >
        Add folder
      </button>
    </div>
  </NotefulForm>
</section>
`;
